### DISCUSSION TITLE
AppStream Screenshot for Terminal Application?

### DISCUSSION THREAD

**Discussion by kuiperzone** [AppImage/AppImageKit#1257] [Q&A] [answered]
The AppStream guidelines make it clear that metadata screenshots should be of the application main window.

However, what's the best approach for a terminal application?

I could show a terminal window with the app's --version output, sure, but that would not be very interesting.

I could include no screenshot, but that looks lazy.

Or I could include some appropriate info image or branding, but that goes against the guidance.

Any thoughts?

**Comment by probonopd** [comment, marked as answer]
Don't know whether there is a general recommendation, but I'd just use a screenshot of the first screen of output from `commandlinetool --help`, without window decorations.
